src/main/java/core/basesyntax/operation/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.operation;
```

Comment:
```suggestion
package core.basesyntax.strategy.handler;
```
same for all other handlers